src/features/generic/time-utils.ts

```diff
@@ -56,6 +56,31 @@ namespace TimeUtilities {
       displayInternal: displayInternal,
     }
   }
-};
+
+  export function calculateWorkedTimeAndBillableHours(user: TimeEntryTotalDto | DailyCombinedData){
+
+    // calculates billable hours and adds percentage sign
+    const billableHours = (user.projectTime/user.expected * 100).toFixed(1); 
```

Comment:
Use destructured syntax (const { variable1, variable2 } = object) for user object instead of calling user.something multiple times

Reply:
Done